Sources/Orbit/Components/Heading.swift

```diff
@@ -94,6 +94,34 @@ public extension Heading {
                 case .title6:           return Text.Size.small.value
             }
         }
+
+        var textStyle: Font.TextStyle {
+            switch self {
+                case .display:          return .largeTitle
+                case .displaySubtitle:
+                    if #available(iOS 14.0, *) {
+                        return .title2
+                    } else {
+                        return .headline
+                    }
+                case .title1:           return .title
```

Comment:
I'd probably put these on separate lines to match the cases with availability ifs... this is harder to read IMO.

Reply:
Updated